code/modules/mob/living/carbon/human/species/station/station.dm

```diff
@@ -466,30 +472,39 @@
 	heat_level_1 = 330 //Default 360
 	heat_level_2 = 380 //Default 400
 	heat_level_3 = 600 //Default 1000
-	flags = NO_SCAN | NO_SLIP
+	flags = NO_SLIP | NO_MINOR_CUT
 	spawn_flags = CAN_JOIN | IS_WHITELISTED
 	appearance_flags = HAS_SKIN_COLOR
 	blood_color = "#E6E600" // dark yellow
-	flesh_color = "#E6E600"
-	base_color = "#575757"
-
-	stamina	=	100			  // Long period of sprinting, but relatively low speed gain
-	sprint_speed_factor = 0.55
-	sprint_cost_factor = 0.20
-	stamina_recovery = 1//slow recovery
+	flesh_color = "#330000"
+	base_color = "#330000"
 
+	death_message = "chitters faintly before crumbling to the ground, their eyes dead and lifeless..."
+	halloss_message = "crumbles to the ground, too weak to continue fighting."
 
-	inherent_verbs = list(
-		/mob/living/carbon/human/proc/bugbite //weaker version of gut.
+	list/heat_discomfort_strings = list(
+		"Your blood feels like its boiling in the heat.",
+		"You feel uncomfortably warm.",
+		"Your carapace feels hot as the sun."
+		)
+	list/cold_discomfort_strings = list(
+		"You chitter in the cold.",
+		"You shiver suddenly.",
+		"Your carapace is ice to the touch."
 		)
 
+	stamina	=	100			  // Long period of sprinting, but relatively low speed gain
+	sprint_speed_factor = 0.7
+	sprint_cost_factor = 0.30
+	stamina_recovery = 2//slow recovery
 
 	has_organ = list(
 		"neural socket" =  /obj/item/organ/vaurca/neuralsocket,
+		"filtration bit" = /obj/item/organ/vaurca/filtrationbit,
 		"lungs" =    /obj/item/organ/lungs,
-		"filtration bit" =     /obj/item/organ/vaurca/filtrationbit,
-		"heart" =    /obj/item/organ/heart,
-		"second heart" =    /obj/item/organ/heart,
+		"phoron reserve tank" = /obj/item/organ/vaurca/preserve,
+		"right heart" =    /obj/item/organ/heart,
```

Comment:
You may want a second heart datum. From what I've seen of the debug logs, the "right heart" and "left heart" stuff doesn't work properly.